src/common/components/deck-add-modal/index.tsx

```diff
@@ -215,17 +229,125 @@ const AddColumn = ({ setSelectedValue, onSelect, selectedValue, decks }: any) =>
 
   let type = selectedValue === _t("decks.community") ? communityContentTypes : contentTypes;
 
+  const searchForm = (
```

Comment:
We have to move it to separate component because file too huge now and there is a encapsulated logic